packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -89,6 +89,12 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    */
   ariaLabel?: string;
 
+  /**
+   * Whether or not the clear button is enabled.
+   * @defaultvalue true
```

Comment:
props should be phrased to be opt-in, not out (therefor defaultValue of false). Instead of "clearable" which is the default state, this should be something like "clearHidden" so that your code looks like:

```
<SearchBox clearHidden />
```

Rather than  

```
<SearchBox clearable={false} />
```

But if all the prop is doing is hiding the clear button, can't we simply do that with styling?

```
    <SearchBox styles={{clearButton: {display: 'none'}}}/>
```

Why increase component weight for something we can already do?